electrum/gui/qt/history_list.py

```diff
@@ -449,14 +440,12 @@ class Columns(MyTreeView.BaseColumnsEnum):
         FIAT_ACQ_PRICE = enum.auto()
         FIAT_CAP_GAINS = enum.auto()
         TXID = enum.auto()
-        SHORT_ID = enum.auto()  # ~SCID
 
     filter_columns = [
         Columns.STATUS,
         Columns.DESCRIPTION,
         Columns.AMOUNT,
         Columns.TXID,
-        Columns.SHORT_ID,
```

Comment:
Indeed the SHORT_ID column is never shown. It was added for this filtering functionality exclusively, in https://github.com/spesmilo/electrum/commit/f2dc651c9b0176a06dcaa53e354784a963ebcf65.

E.g. if you are testing with the wallet from https://github.com/spesmilo/electrum/issues/6625#issuecomment-724912330, press Ctrl+F and type `1326030x59`. That filters the list to the tx at block height `1326030`, pos in block `59`.

If it's not that expensive to compute, I think it's a nice (though certainly niche) feature.